cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,92 @@
+- ../stable/cockroachcloud-frequently-asked-questions.html
+---
+
+This page answers the frequently asked questions about CockroachCloud Free (beta). For answers to frequently asked questions about the paid version of CockroachCloud, see [CockroachCloud FAQs](frequently-asked-questions.html).
+
+## General
+
+### What is CockroachCloud Free?
+
+CockroachCloud Free delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that also removes the friction of initial cluster sizing and auto-scales based on your application traffic. While we have eliminated the concept of nodes for you, there will be an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster.
+
+### What can I use CockroachCloud Free for?
+
+CockroachCloud Free can be used for proofs-of-concept, toy programs, or to use while completing [Cockroach University](https://www.cockroachlabs.com/cockroach-university/).
+
+For examples of applications that use CockroachCloud Free, check out the following [Hack the North](https://hackthenorth.com/) projects:
+
+- [flock](https://devpost.com/software/flock-figure-out-what-film-to-watch-with-friends)
+- [mntr.tech](https://devpost.com/software/mntr-tech)
+- [curbshop.online](https://devpost.com/software/curbshop-online)
+
+### What are the limitations of CockroachCloud Free?
+
+CockroachCloud Free is currently in beta and as such, it is not suitable for production use. Specifically, there are capabilities we are still working on enabling, such as the ability to enable backups, to import data, and no-downtime upgrades to a paid tier. These capabilities and more are coming soon!
+
+### Can I use a specific region in AWS or GCP?
+
+At this time, CockroachCloud Free is only available in the GCP `us-central-1` region.
+
+### Do I have to pay for CockroachCloud Free?
+
+CockroachCloud Free is free forever. However, it is currently not recommended for production use. For production applications, we recommend using the paid version of [CockroachCloud](create-your-cluster.html).
+
+### How do I connect to my cluster?
+
+To connect to a cluster, download the CA certificate, and then generate a connection string or parameters. You can use this information to connect to your cluster through the CockroachDB SQL client or a Postgres-compatible driver or ORM. For more details, see [Connect to Your CockroachCloud Cluster](connect-to-your-cluster.html).
+
+## Security
+
+### Is my cluster secure?
+
+Yes. We create individual sub-accounts and VPCs for each cluster within the cloud provider. These VPCs are firewalled from each other and any other outside connection, unless allowlisted for SQL and Web UI ports.
+
+The allowlist is comprised of IP addresses that you provide to us, and is an additional layer of protection for your cluster. Connections will only be accepted if they come from an allowlisted IP address, which protects against both compromised passwords and any potential bugs in the server.
```

Comment:
Not sure this part is true for free tier, there is no allowlist for free from a customer pov.